webapps/webapp-template/src/app/AppHandler.tsx

```
@@ -0,0 +1,78 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+
+import { useEffect, useState } from "react";
+
+import Layout from "@layout/Layout";
+import Error from "@layout/pages/404";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)